tao/x11/taox11.mpc

```diff
@@ -160,7 +160,7 @@ project(taox11) : taox11_defaults, taolib {
     anyinsert_adapter.h
     CurrentC.h
     orb_typesC.h
-    InterfaceDefC.h
+//     InterfaceDefC.h
```

Comment:
Please remove files that don't exist instead of commenting them out

Reply:
ok